app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Section/AdminMediaGalleryCatalogUiCategoryGridSection.xml

```diff
@@ -9,14 +9,17 @@
 <sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdminMediaGalleryCatalogUiCategoryGridSection">
+        <!--Search by category name element-->
+        <element name="search" type="input" selector=".admin__data-grid-header[data-bind='afterRender: \$data.setToolbarNode'] input[placeholder='Search by category name']"/>
```

Comment:
Please do not use `data-bind` in a selector, that is part of the functionality implementation that can be changed